### PR TITLE
Fix for various issues running integration tests against a VC cluster

### PR DESCRIPTION
**Pull request by corrieb**
Fixes #

### CODE REVIEW

File: tests/resources/Util.robot [hunk 1]
```diff
@@ -75,7 +76,7 @@ Install VIC Appliance To Test Server
 
     # Install the VCH now
     Log To Console  \nInstalling VCH to test server...
-    ${output}=  Run VIC Machine Command  ${certs}  ${vol}  network  'VM Network'
+    ${output}=  Run VIC Machine Command  ${certs}  ${vol}  ${bridge}  ${external}
```
Comment (mhagen-vmware) [inline tests/resources/Util.robot]: we need to actually push this all the way up to line 70 as well, I could have sworn I made that change already but not sure what the heck happened...

Comment (corrieb) [inline tests/resources/Util.robot]: Maybe what I've done now is what you were suggesting but in a different way - it makes sure the env vars are set regardless. More ugly, but less error prone

**Comment by mhagen-vmware** [comment]
lgtm

**Comment by mhagen-vmware** [comment]
I will push another PR out to make the other change I wanted.

**Comment by mdubya66** [comment]
lgtm

**Comment by mhagen-vmware** [comment]
https://github.com/vmware/vic/issues/1978

**Comment by corrieb** [comment]
ping

**Pull request merged**